presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSessionProperties.java

```diff
@@ -369,6 +370,10 @@ public HiveSessionProperties(
                         PARQUET_OPTIMIZED_WRITER_ENABLED,
                         "Experimental: Enable optimized writer",
                         parquetWriterConfig.isParquetOptimizedWriterEnabled(),
+                        false),
+               booleanProperty(PURGE_TABLE_DATA_ON_DROP,
+                        "Skip trash when dropping tables",
```

Comment:
Since this is applicable to thrift metastore only, it should be configurable only when using that.
Also, not sure it's safe to have this as a session property at all. Maybe a configuration toggle will be sufficient?

@electrum do you have thoughts on that?

Reply:
https://github.com/prestosql/presto/pull/4263#issuecomment-653644082